src/app/parcels/parcelsTable/ParcelsPage.cy.tsx

```diff
@@ -103,11 +98,6 @@ describe("Parcels Page", () => {
             expect(formatDatetimeAsDate(null)).to.eq("-");
         });
 
-        it("datetimeToPackingTimeLabel()", () => {
-            expect(datetimeToPackingTimeLabel("2023-08-04T08:30:00+00:00")).to.eq("AM");
-            expect(datetimeToPackingTimeLabel("2023-08-04T13:30:00+00:00")).to.eq("PM");
-        });
```

Comment:
This test is no longer needed as the datetimeToPackingTimeLabel function is no longer used